website/content/docs/setup/control-plane-security/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Control-plane Security
+linktitle: Control-plane Security
+weight: 30
+---
+
+Upon {{< name >}} deployment [Kubernetes NetworkPolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) resource is applied to ensure that the connector ingress is limited to traffic only from workloads within the control-plane namespace. 
+In addition, Istio authentication policy can be applied upon installation of {{< name >}} to ensure mutual TLS between the pilot and connectors. 
+Istio authentication policy is not part of the default {{< name >}} deployment but it can be applied by running the manager control-plane-security configuration as described in the manager README [config section](https://{{< github_base >}}/{{< github_repo >}}/tree/master/manager#config).
```

Comment:
Please add concrete steps in a bash block for deploying these Istio stuff. 
```bash
command 1
command 1
```

Also, it's not an authorization policy. Just say Istio resources.